importScripts/sampleRecords/library.source.json

```diff
@@ -0,0 +1,738 @@
+  "WherePublished": "Perth",
+  "Subject": "Nyungar (Australian people); Aborigines - Albany region -  Language -  culture;",
+  "Illustrations": "1",
+  "Maps": "1",
+  "Marks": "206p. Illus., maps, photos",
+  "Pages": "206",
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateDue": "2002-07-27",
+  "DateOut": "2002-06-29",
+  "Borrower": "NOT FOR LOAN",
+  "Value": "0"
+ },
+ {
+  "PublicationID": "12566",
+  "Title": "Mission cottage restoration (New Norcia Studies, No.17, 2009, pp.46-54)",
+  "LegacyID": "12566",
+  "Series": "0",
+  "Authors": "Bremen, Ingrid H van; Campbell, Robin McK",
+  "LastName": "Bremen",
+  "FirstNames": "Ingrid H van",
+  "OtherAuthors": "Campbell, Robin McK",
+  "HomeLocation": "LIBRARY",
+  "LastEditBy": "JENM",
+  "LastEditDate": "2015-03-19 00:00:00",
+  "LibraryNo": "J   NEW",
+  "PublicationType": "Journal article",
+  "PublicationYear": "2009",
+  "Publisher": "Benedictine Community of New Norcia",
+  "WherePublished": "New Norcia",
+  "Subject": "Cottages - New Norcia - Conservation and restoration",
+  "CurrentLocation": "ANNEXE",
+  "Illustrations": "1",
+  "Maps": "0",
+  "Pages": "0",
+  "Rare": "0",
+  "Restrictions": "NOT  FOR  LOAN",
+  "DateDue": "2011-06-02",
+  "DeweyNumber": "No. 17",
+  "Borrower": "NOT  FOR  LOAN",
+  "Value": "0"
+ },
+ {
+  "PublicationID": "13738",
+  "Title": "Extracts from Western Australian Newspapers 1870-1888c",
+  "LegacyID": "13738",
+  "Series": "0",
+  "Authors": "Moss, W.E.C.",
+  "LastName": "Moss",
+  "FirstNames": "W.E.C.",
+  "HomeLocation": "LIBRARY",
+  "LastEditBy": "MCCJ",
+  "LastEditDate": "2012-05-24 00:00:00",
+  "LibraryNo": "J  H/S Series 505",
+  "Notes": "HELD IN BATTYE LIBRARY",
+  "PublicationType": "\r\nJournal article",
+  "PublicationYear": "1888",
+  "Subject": "Press coverage - Western Australia; Clippings (Newspapers)",
+  "CurrentLocation": "BATTYE LIBRARY",
+  "Illustrations": "0",
+  "Maps": "0",
+  "Marks": "128p.",
+  "Pages": "128",
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateOut": "2012-05-24",
+  "Borrower": "NOT FOR LOAN",
+  "Value": "0"
+ },
+ {
+  "PublicationID": "139",
+  "Title": "The ways of life",
+  "LegacyID": "139",
+  "Series": "0",
+  "Edition": "3rd.",
+  "Authors": "Bain, Evan (Ernie)",
+  "LastName": "Bain",
+  "FirstNames": "Evan (Ernie)",
+  "HomeLocation": "LIBRARY",
+  "LastEditBy": "JILLM",
+  "LastEditDate": "2006-05-03 00:00:00",
+  "LibraryNo": "BAI",
+  "PublicationType": "Book",
+  "PublicationYear": "1976",
+  "Publisher": "Elder Smith",
+  "WherePublished": "Perth",
+  "Subject": "Bain, Evan - Autobiography; Drovers",
+  "Illustrations": "1",
+  "Maps": "0",
+  "Marks": "viii, 130p. Illus., photos.",
+  "Pages": "130",
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateDue": "2002-02-21",
+  "DateOut": "2002-01-24",
+  "Borrower": "NOT FOR LOAN",
+  "Value": "0"
+ },
+ {
+  "PublicationID": "14118",
+  "Title": "Grain pool history",
```

Comment:
Actually I think it's better if these files go undoctored, can you regenerate this and update it?